### DISCUSSION TITLE
Bug: Cannot read properties of undefined (reading 'value')

### DISCUSSION THREAD

**Discussion by AutumnDeSea** [statelyai/xstate#4578] [General]
## my npm version ：
 
 "@xstate/vue": "^3.0.0",
  "xstate": "^5.2.1"
  
## my code  is your example code: 
```js
<script setup>
import { useMachine } from '@xstate/vue';
import { createMachine } from 'xstate';

const toggleMachine = createMachine({
  id: 'toggle',
  initial: 'inactive',
  states: {
    inactive: {
      on: { TOGGLE: 'active' }
    },
    active: {
      on: { TOGGLE: 'inactive' }
    }
  }
});

const { state, send } = useMachine(toggleMachine);
</script>

<template>
  <button @click="send('TOGGLE')">
    {{
      state.value === 'inactive'
        ? 'Click to activate'
        : 'Active! Click to deactivate'
    }}
  </button>
</template>
```
## my console panel happed some error:
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'value')
    at Proxy._sfc_render (HomeView.vue:24:13)

**Comment by davidkpiano** [comment]
Very sorry, I'm in the process of updating the docs for the latest `@xstate/vue` 😅 

Here is a quick migration guide: https://stately.ai/docs/migration#xstatevue

The `useMachine` hook now returns `{ snapshot, send, actor }` instead of `{ state, send, service }`, to keep the naming consistent with the rest of XState.
